packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -636,6 +637,7 @@ export const makeInnerVault = (
     updateUiState();
     clientSeat.exit();
 
+    // TODO get rid of strings?
```

Comment:
if you happen to make another commit on this branch, this could be a good issue for the TODO https://github.com/Agoric/agoric-sdk/issues/4360